From 3d045a240bb32b66e15401bf89eff5b980420b24 Mon Sep 17 00:00:00 2001
From: Mark Reynolds <mreynolds@redhat.com>
Date: Fri, 3 Nov 2017 12:18:26 -0400
Subject: [PATCH] Ticket 48393 - fix copy and paste error

Description:  Copy and paste error when validating repl agmt

https://pagure.io/389-ds-base/issue/48393

Reviewed by: mreynolds(one line commit rule)

(cherry picked from commit 431647039c5e6d860d8866542050d456f69bb600)
---
 ldap/servers/plugins/replication/repl5_agmt.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ldap/servers/plugins/replication/repl5_agmt.c b/ldap/servers/plugins/replication/repl5_agmt.c
index 78fb91ae6..ee396c8ef 100644
--- a/ldap/servers/plugins/replication/repl5_agmt.c
+++ b/ldap/servers/plugins/replication/repl5_agmt.c
@@ -339,7 +339,7 @@ agmt_new_from_entry(Slapi_Entry *e)
     ra->flowControlWindow = DEFAULT_FLOWCONTROL_WINDOW;
     if ((val = slapi_entry_attr_get_charptr(e, type_nsds5ReplicaFlowControlWindow))){
         int64_t flow;
-        if (repl_config_valid_num(type_nsds5ReplicaTimeout, val, 0, INT_MAX, &rc, errormsg, &flow) != 0) {
+        if (repl_config_valid_num(type_nsds5ReplicaFlowControlWindow, val, 0, INT_MAX, &rc, errormsg, &flow) != 0) {
             goto loser;
         }
         slapi_ch_free_string(&val);
-- 
2.13.6

